app/views/shared/_cancel.html.slim

```diff
@@ -1,4 +1,6 @@
 - cancel = sign_up_or_idv_no_js_link || link
+- decorated_session = decorated_session
+- sp_name = decorated_session ? decorated_session.sp_name : ''
```

Comment:
The general rule we follow is to avoid conditionals in views, which is why we created classes like `SessionDecorator` and `ServiceProviderSessionDecorator`.